music.py

```diff
@@ -0,0 +1,84 @@
+import os
+import collections
+
+from pymongo import MongoClient
```

Comment:
Yeah, I'll work this into the if check to see if they've agreed to the warnings.